### ISSUE TITLE
Error: Package: nagios-plugins-perfsonar-4.3.0-1.el7.x86_64 (perfSONAR)

### ISSUE THREAD

**Issue by VicVV3** [perfsonar/nagios#20]
Dear team,

When we trying to install Perfsonar in a empty Centos 7 following the guide : https://docs.perfsonar.net/install_centos.html
We get the following error: 

```
yum install perfsonar-centralmanagement
[...]
Error: Package: nagios-plugins-perfsonar-4.3.0-1.el7.x86_64 (perfSONAR)
           Requires: python-pscheduler
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

Can you help us?

Kind regards,

Victoria

**Comment by laeti-tia** [comment]
Thanks for your report. We're aware of this issue and it will be fixed shortly in an upcoming 4.3.2 release.

**Comment by laeti-tia** [comment]
This is solved in 4.3.2, please re-open again if it doesn't work for you.
